.swift-version

```diff
@@ -0,0 +1 @@
+2.3
```

Comment:
I believe the swift version message is [only displayed as a hint when there's any warning](https://github.com/CocoaPods/CocoaPods/blob/cae98c5386f7d80d6fe065c3f081ce130628151a/lib/cocoapods/validator.rb#L146) and if you fix the rest of the warnings, it'll go away